src/utils.ts

```diff
@@ -295,6 +295,15 @@ export function dateToString(date: Date | string) {
   });
 }
 
+/**
+ * Performs a case-insensitive comparison of two strings
+ * @param a
+ * @param b
+ */
+export function ciCompare(a: string, b: string): number {
```

Comment:
Would you mind changing this function name to `caseInsensitiveCompare` or something similar? Using just "ci" might be a bit confusing when seeing this function used throughout the code. Thanks!